Mathlib/Analysis/Fourier/FourierTransformDeriv.lean

```diff
@@ -769,7 +769,7 @@ lemma pow_mul_norm_iteratedFDeriv_fourier_le
     gcongr
     exact norm_innerSL_le _
   rcases eq_or_ne w 0 with rfl | hw
-  · simp [hn]
+  · simp only [norm_zero, ne_eq, hn, not_false_eq_true, zero_pow, zero_mul, ge_iff_le]
```

Comment:
You've allowed most other cases with `positivity` - should we leave this one alone for now as well?